src/app/modules/MainArticlesScreen/MainArticlesScreen.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { PureComponent } from 'react';
+import { Button, Picker, Text, View } from 'react-native';
+import { NavigationStackScreenProps } from 'react-navigation-stack';
+import { WrappedComponentProps } from 'react-intl';
+import { v1 } from 'uuid';
+
+import {
+  ArticleData,
+  topicValues,
+  timeValues,
+  sortValues,
+} from '@modules/MainArticlesScreen/namespace';
```

Comment:
why modules, not pages, like we have in web version of CRA? let's keep things consistent, please